app/app.go

```diff
@@ -0,0 +1,199 @@
+package app
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"io"
+	"strings"
+
+	"github.com/datatrails/go-datatrails-logverification/logverification/app"
+)
+
+/**
+ * Merklelog related APP contents
+ *
+ * An APP in the context of merklelog is an interface that commits log entries.
+ *
+ * Apps include:
+ *  - assetsv2
+ *  - eventsv1
+ */
+
+const (
+	AssetsV2AppDomain = byte(0)
+	EventsV1AppDomain = byte(1)
+)
+
+var (
+	ErrNoJsonGiven = errors.New("no json given")
+)
+
+// AppDataToVerifiableLogEntries converts the app data to verifiable log entries
+func AppDataToVerifiableLogEntries(appData []byte, logTenant string) ([]app.AppEntry, error) {
```

Comment:
Its not immediately clear what `appData` means. It would be helpful to define this in the comment

Reply:
will add a comment :)

Reply:
done!